src/@types/components/commonTypes.tsx

```diff
@@ -288,3 +289,17 @@ export interface BarChartCompProps {
 export interface ChartProps {
   type: string;
 }
+
+export enum SideBarPositionEnums {
+  RIGHT = "right",
+  LEFT = "left",
+}
+
+export interface CustomSideBarProps extends SidebarProps {
+  visible: boolean;
+  setVisible?: () => void;
+  position: SideBarPositionEnums;
+  title?: string;
+  children?: React.ReactNode;
+  themeClasses?: string;
```

Comment:
visible, position, children are already part of SidebarProps 
Any reason for redeclaring again ?

Reply:
Removed visible, position & children props from CustomSidebarProps